src/dataflow/analysis/tests/flow-graph-test.ts

```diff
@@ -118,7 +118,7 @@ describe('FlowGraph', () => {
           input <- s
     `);
     assert.lengthOf(graph.edges, 1);
-    const claim = graph.edges[0].claim;
+    const claim = graph.edges[0].claims[0];
```

Comment:
Might be worth asserting the length of this second array too

Reply:
Done.